tests/test_special_ops.py

```diff
@@ -493,8 +502,8 @@ def test_upsample_nearest2d(dtype, shape, scale):
 @pytest.mark.parametrize("start", [0, 1, 3])
 @pytest.mark.parametrize("step", [1, 2, 5])
 @pytest.mark.parametrize("end", [128, 256, 1024])
-@pytest.mark.parametrize("dtype", FLOAT_DTYPES + ALL_INT_DTYPES + [None])
-@pytest.mark.parametrize("device", ["cuda", None])
+@pytest.mark.parametrize("dtype", [torch.float16, torch.float32] + ALL_INT_DTYPES + [None])
```

Comment:
doesn't support torch.bfloat16?

Reply:
not tested yet, shall we confirm later and this time makes it still?